docs/project/2024/s_m.takdeer/index.md

```diff
@@ -0,0 +1,85 @@
+
+## VitalTrack: Wearable Fitness and Health Monitor
+A wearable device that tracks steps, heart rate, and location, with data visualization and smartphone integration.
+
+
+:::info
+
+Author: TAKDEER S M
```

Comment:
Please write your full name.

Reply:
This is my name in all of my id cards and documents

Reply:
S M? Ok